src/Poser.php

```diff
@@ -13,12 +13,9 @@ class Poser
      *
      * @param $renders
      */
-    public function __construct($renders)
+    public function __construct(array $renders)
```

Comment:
This could be a BC.
cc @JellyBellyDev @garak

Reply:
Who cares? 😜

Reply:
We can create a v3?

Reply:
If someone uses this method it's a BC.
I have changed also in the README file how to use this class.

Reply:
I see! Please can you change also the CHANGELOG.md file?

Reply:
Done